Makefile

```diff
@@ -1,131 +0,0 @@
-.PHONY: all clean validate-spv validate-msl validate-glsl validate-dot validate-wgsl validate-hlsl-dxc validate-hlsl-fxc
-.SECONDARY: boids.metal quad.metal
-SNAPSHOTS_BASE_IN=tests/in
-SNAPSHOTS_BASE_OUT=tests/out
-
-all:
-	cargo fmt
-	cargo test --all-features --workspace
-	cargo clippy --all-features --workspace -- -D warnings
-
-clean:
-	rm *.metal *.air *.metallib *.vert *.frag *.comp *.spv
-
-bench:
-	#rm -Rf target/criterion
-	cargo bench
-
-%.metal: $(SNAPSHOTS_BASE_IN)/%.wgsl $(wildcard src/*.rs src/**/*.rs examples/*.rs)
-	cargo run --features wgsl-in,msl-out -- $< $@
-
-%.air: %.metal
-	xcrun -sdk macosx metal -c $< -mmacosx-version-min=10.11
-
-%.metallib: %.air
-	xcrun -sdk macosx metallib $< -o $@
-
-%.dot: $(SNAPSHOTS_BASE_IN)/%.wgsl $(wildcard src/*.rs src/front/wgsl/*.rs src/back/dot/*.rs bin/naga.rs)
-	cargo run --features wgsl-in,dot-out -- $< $@
-
-%.png: %.dot
-	dot -Tpng $< -o $@
```

Comment:
CC @jimblandy, @cwfitzgerald.

Reply:
I'm indifferent.

Reply:
Since nobody has felt strongly about this here or [in Matrix](https://matrix.to/#/!fjjkgHFcwtkREywzfk:matrix.org/$8fUNjQx5Pm-Pgz6fQH2CGRKeS8RmW-BOMlaQ4_qQQeE?via=matrix.org&via=mozilla.org&via=anontier.nl), I'm gonna mark this as `Resolved`/"won't fix".